src/components/PresetEditor/PresetEditorTurn.tsx

```diff
@@ -10,8 +10,7 @@ interface IProps {
 }
 
 export const PresetEditorTurn = ({index, turn, onValueChange}: IProps) =>
-    <div className="columns is-mobile has-background-light" key={'turn-' + index}
-         style={{borderBottom: '1px solid #ddd'}}>
+    <div className="columns is-mobile preset-editor-row" key={'turn-' + index}>
```

Comment:
That seems to be a bit darker than the previous background colour, which makes the checkboxes stand out. Is that on purpose?
![image](https://user-images.githubusercontent.com/3027817/87424208-dd53fc80-c5db-11ea-84ec-3752a211d012.png)

Reply:
Similar issue with the dark theme
![image](https://user-images.githubusercontent.com/3027817/87424319-08d6e700-c5dc-11ea-901d-822a2248bf2c.png)

Reply:
This is not intentional. It is a side effect of some other style I changed. I will correct it.

Reply:
Fixed.